packages/verified-fetch/src/utils/get-tar-stream.ts

```diff
@@ -28,7 +27,7 @@ function toHeader (file: UnixFSEntry): Partial<TarEntryHeader> & { name: string
 
 function toTarImportCandidate (entry: UnixFSEntry): TarImportCandidate {
   if (!EXPORTABLE.includes(entry.type)) {
-    throw new CodeError('Not a UnixFS node', 'ERR_NOT_UNIXFS')
+    throw new Error('Not a UnixFS node')
```

Comment:
You might want to use named errors for better DX here and elsewhere in this PR?

`@helia/unixfs` exports a `NotUnixFSError`, for example:

```ts
import { NotUnixFSError }  from '@helia/unixfs/errors'
```